### DISCUSSION TITLE
Alpine error

### DISCUSSION THREAD

**Discussion by MJ0KER** [wire-elements/modal#402] [General]
use @livewire('wire-elements-modal') and error

bootstrap.js:10  Alpine Expression Error: Cannot read properties of undefined (reading 'get')

Expression: "function(){var t=this;this.modalWidth=this.getActiveComponentModalAttribute("maxWidthClass"),this.listeners.push(Livewire.on("closeModal",(function(e){var o,n,i;t.closeModal(null!==(o=null==e?void 0:e.force)&&void 0!==o&&o,null!==(n=null==e?void 0:e.skipPreviousModals)&&void 0!==n?n:0,null!==(i=null==e?void 0:e.destroySkipped)&&void 0!==i&&i)}))),this.listeners.push(Livewire.on("activeModalComponentChanged",(function(e){var o=e.id;t.setActiveModalComponent(o)})))}"

**Comment by lrljoe** [comment]
Sounds like the order you're loading bootstrap and livewire isn't quite right.

Are you using livewire 2 or 3, are you injecting or bundling?

What does your layout file look like?
